docs/src/modules/components/AppFrame.js

```diff
@@ -93,7 +95,10 @@ const styles = theme => ({
     boxShadow: 'none',
   },
   language: {
-    margin: theme.spacing(0, 1, 0, 0.5),
+    margin: theme.spacing(0, 0.5, 0, 1),
+  },
+  iconButton: {
```

Comment:
Do think that we could rely on a CSS selector, like `>  *`? It could remove the need to apply a class name to each child.

Reply:
Alternatively, what about keeping 0 margins, reducing the icon size to fit on mobile?
![Capture d’écran 2019-09-19 à 13 41 35](https://user-images.githubusercontent.com/3165635/65241191-431e4780-dae3-11e9-8751-60255f4a6cff.png)